macros/unpack/get_source_values.sql

```diff
@@ -5,11 +5,22 @@
 {%- macro default__get_source_values() -%}
 
     {%- if execute -%}
+    {% set re = modules.re %}
     {%- set nodes_list = graph.sources.values() -%}
     {%- set values = [] -%}
 
     {%- for node in nodes_list -%}
 
+        {%- set ns = namespace(exclude=false) -%}
+        {%- set node_package_path = node.package_name ~ ":" ~ node.original_file_path | replace("\\","\\\\") ~ ":" ~ node.name -%}
```

Comment:
do we need to worry about the windows slash situation here?

Reply:
Nah, `replace("\\","\\\\")` is actually relevant only for Windows already and doesn't do anything on Mac.